CRITICAL - 2021-09-19 02:22:55 --> Unable to connect to the database.
#0 C:\xampp\htdocs\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\app\Filters\CheckLogin.php(0): CodeIgniter\Database\BaseConnection->query('select token  F...')
#2 C:\xampp\htdocs\system\Filters\Filters.php(173): App\Filters\checkLogin->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\system\CodeIgniter.php(382): CodeIgniter\Filters\Filters->run('/', 'before')
#4 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-09-19 06:55:20 --> Unable to connect to the database.
#0 C:\xampp\htdocs\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\app\Filters\CheckLogin.php(0): CodeIgniter\Database\BaseConnection->query('select token  F...')
#2 C:\xampp\htdocs\system\Filters\Filters.php(173): App\Filters\checkLogin->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\system\CodeIgniter.php(382): CodeIgniter\Filters\Filters->run('/', 'before')
#4 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-09-19 07:32:20 --> Unable to connect to the database.
#0 C:\xampp\htdocs\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\app\Filters\CheckLogin.php(0): CodeIgniter\Database\BaseConnection->query('select token  F...')
#2 C:\xampp\htdocs\system\Filters\Filters.php(173): App\Filters\checkLogin->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\system\CodeIgniter.php(382): CodeIgniter\Filters\Filters->run('112.124.42.80:6...', 'before')
#4 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-09-19 07:33:51 --> Unable to connect to the database.
#0 C:\xampp\htdocs\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\app\Filters\CheckLogin.php(0): CodeIgniter\Database\BaseConnection->query('select token  F...')
#2 C:\xampp\htdocs\system\Filters\Filters.php(173): App\Filters\checkLogin->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\system\CodeIgniter.php(382): CodeIgniter\Filters\Filters->run('110.242.68.4', 'before')
#4 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-09-19 08:41:41 --> Unable to connect to the database.
#0 C:\xampp\htdocs\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\app\Filters\CheckLogin.php(0): CodeIgniter\Database\BaseConnection->query('select token  F...')
#2 C:\xampp\htdocs\system\Filters\Filters.php(173): App\Filters\checkLogin->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\system\CodeIgniter.php(382): CodeIgniter\Filters\Filters->run('/', 'before')
#4 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-09-19 09:20:50 --> Unable to connect to the database.
#0 C:\xampp\htdocs\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\app\Filters\CheckLogin.php(0): CodeIgniter\Database\BaseConnection->query('select token  F...')
#2 C:\xampp\htdocs\system\Filters\Filters.php(173): App\Filters\checkLogin->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\system\CodeIgniter.php(382): CodeIgniter\Filters\Filters->run('/', 'before')
#4 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-09-19 09:20:51 --> Unable to connect to the database.
#0 C:\xampp\htdocs\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\app\Filters\CheckLogin.php(0): CodeIgniter\Database\BaseConnection->query('select token  F...')
#2 C:\xampp\htdocs\system\Filters\Filters.php(173): App\Filters\checkLogin->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\system\CodeIgniter.php(382): CodeIgniter\Filters\Filters->run('/favicon.ico', 'before')
#4 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-09-19 09:26:06 --> Unable to connect to the database.
#0 C:\xampp\htdocs\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\app\Filters\CheckLogin.php(0): CodeIgniter\Database\BaseConnection->query('select token  F...')
#2 C:\xampp\htdocs\system\Filters\Filters.php(173): App\Filters\checkLogin->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\system\CodeIgniter.php(382): CodeIgniter\Filters\Filters->run('/', 'before')
#4 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-09-19 09:26:10 --> Unable to connect to the database.
#0 C:\xampp\htdocs\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\app\Filters\CheckLogin.php(0): CodeIgniter\Database\BaseConnection->query('select token  F...')
#2 C:\xampp\htdocs\system\Filters\Filters.php(173): App\Filters\checkLogin->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\system\CodeIgniter.php(382): CodeIgniter\Filters\Filters->run('/', 'before')
#4 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-09-19 10:16:54 --> Unable to connect to the database.
#0 C:\xampp\htdocs\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\app\Filters\CheckLogin.php(0): CodeIgniter\Database\BaseConnection->query('select token  F...')
#2 C:\xampp\htdocs\system\Filters\Filters.php(173): App\Filters\checkLogin->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\system\CodeIgniter.php(382): CodeIgniter\Filters\Filters->run('/', 'before')
#4 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-09-19 11:45:17 --> Unable to connect to the database.
#0 C:\xampp\htdocs\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\app\Filters\CheckLogin.php(0): CodeIgniter\Database\BaseConnection->query('select token  F...')
#2 C:\xampp\htdocs\system\Filters\Filters.php(173): App\Filters\checkLogin->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\system\CodeIgniter.php(382): CodeIgniter\Filters\Filters->run('/', 'before')
#4 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-09-19 13:13:12 --> Unable to connect to the database.
#0 C:\xampp\htdocs\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\app\Filters\CheckLogin.php(0): CodeIgniter\Database\BaseConnection->query('select token  F...')
#2 C:\xampp\htdocs\system\Filters\Filters.php(173): App\Filters\checkLogin->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\system\CodeIgniter.php(382): CodeIgniter\Filters\Filters->run('/', 'before')
#4 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-09-19 13:13:17 --> Unable to connect to the database.
#0 C:\xampp\htdocs\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\app\Filters\CheckLogin.php(0): CodeIgniter\Database\BaseConnection->query('select token  F...')
#2 C:\xampp\htdocs\system\Filters\Filters.php(173): App\Filters\checkLogin->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\system\CodeIgniter.php(382): CodeIgniter\Filters\Filters->run('/favicon.ico', 'before')
#4 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-09-19 13:15:46 --> Unable to connect to the database.
#0 C:\xampp\htdocs\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\app\Filters\CheckLogin.php(0): CodeIgniter\Database\BaseConnection->query('select token  F...')
#2 C:\xampp\htdocs\system\Filters\Filters.php(173): App\Filters\checkLogin->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\system\CodeIgniter.php(382): CodeIgniter\Filters\Filters->run('/favicon.ico', 'before')
#4 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-09-19 13:22:23 --> Unable to connect to the database.
#0 C:\xampp\htdocs\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\app\Filters\CheckLogin.php(0): CodeIgniter\Database\BaseConnection->query('select token  F...')
#2 C:\xampp\htdocs\system\Filters\Filters.php(173): App\Filters\checkLogin->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\system\CodeIgniter.php(382): CodeIgniter\Filters\Filters->run('/', 'before')
#4 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-09-19 13:22:24 --> Unable to connect to the database.
#0 C:\xampp\htdocs\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\app\Filters\CheckLogin.php(0): CodeIgniter\Database\BaseConnection->query('select token  F...')
#2 C:\xampp\htdocs\system\Filters\Filters.php(173): App\Filters\checkLogin->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\system\CodeIgniter.php(382): CodeIgniter\Filters\Filters->run('/favicon.ico', 'before')
#4 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-09-19 13:22:28 --> Unable to connect to the database.
#0 C:\xampp\htdocs\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\app\Filters\CheckLogin.php(0): CodeIgniter\Database\BaseConnection->query('select token  F...')
#2 C:\xampp\htdocs\system\Filters\Filters.php(173): App\Filters\checkLogin->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\system\CodeIgniter.php(382): CodeIgniter\Filters\Filters->run('/', 'before')
#4 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-09-19 13:22:29 --> Unable to connect to the database.
#0 C:\xampp\htdocs\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\app\Filters\CheckLogin.php(0): CodeIgniter\Database\BaseConnection->query('select token  F...')
#2 C:\xampp\htdocs\system\Filters\Filters.php(173): App\Filters\checkLogin->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\system\CodeIgniter.php(382): CodeIgniter\Filters\Filters->run('/favicon.ico', 'before')
#4 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-09-19 13:32:00 --> syntax error, unexpected ''checkLogin'' (T_CONSTANT_ENCAPSED_STRING), expecting ']'
#0 C:\xampp\htdocs\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
#3 C:\xampp\htdocs\system\Config\Services.php(322): spl_autoload_call('Config\\Filters')
#4 C:\xampp\htdocs\system\Config\BaseService.php(123): CodeIgniter\Config\Services::filters(NULL, false)
#5 C:\xampp\htdocs\system\Config\Services.php(317): CodeIgniter\Config\BaseService::getSharedInstance('filters', NULL)
#6 C:\xampp\htdocs\system\CodeIgniter.php(367): CodeIgniter\Config\Services::filters()
#7 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2021-09-19 13:32:02 --> syntax error, unexpected ''checkLogin'' (T_CONSTANT_ENCAPSED_STRING), expecting ']'
#0 C:\xampp\htdocs\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
#3 C:\xampp\htdocs\system\Config\Services.php(322): spl_autoload_call('Config\\Filters')
#4 C:\xampp\htdocs\system\Config\BaseService.php(123): CodeIgniter\Config\Services::filters(NULL, false)
#5 C:\xampp\htdocs\system\Config\Services.php(317): CodeIgniter\Config\BaseService::getSharedInstance('filters', NULL)
#6 C:\xampp\htdocs\system\CodeIgniter.php(367): CodeIgniter\Config\Services::filters()
#7 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2021-09-19 13:32:33 --> honeypotcheckLogin filter must have a matching alias defined.
#0 C:\xampp\htdocs\system\Filters\Filters.php(150): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('honeypotcheckLo...')
#1 C:\xampp\htdocs\system\CodeIgniter.php(382): CodeIgniter\Filters\Filters->run('/home', 'before')
#2 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-09-19 13:32:35 --> honeypotcheckLogin filter must have a matching alias defined.
#0 C:\xampp\htdocs\system\Filters\Filters.php(150): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('honeypotcheckLo...')
#1 C:\xampp\htdocs\system\CodeIgniter.php(382): CodeIgniter\Filters\Filters->run('/favicon.ico', 'before')
#2 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-09-19 13:37:32 --> syntax error, unexpected 'except' (T_STRING), expecting ']'
#0 C:\xampp\htdocs\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
#3 C:\xampp\htdocs\system\Config\Services.php(322): spl_autoload_call('Config\\Filters')
#4 C:\xampp\htdocs\system\Config\BaseService.php(123): CodeIgniter\Config\Services::filters(NULL, false)
#5 C:\xampp\htdocs\system\Config\Services.php(317): CodeIgniter\Config\BaseService::getSharedInstance('filters', NULL)
#6 C:\xampp\htdocs\system\CodeIgniter.php(367): CodeIgniter\Config\Services::filters()
#7 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2021-09-19 13:37:33 --> syntax error, unexpected 'except' (T_STRING), expecting ']'
#0 C:\xampp\htdocs\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
#3 C:\xampp\htdocs\system\Config\Services.php(322): spl_autoload_call('Config\\Filters')
#4 C:\xampp\htdocs\system\Config\BaseService.php(123): CodeIgniter\Config\Services::filters(NULL, false)
#5 C:\xampp\htdocs\system\Config\Services.php(317): CodeIgniter\Config\BaseService::getSharedInstance('filters', NULL)
#6 C:\xampp\htdocs\system\CodeIgniter.php(367): CodeIgniter\Config\Services::filters()
#7 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2021-09-19 13:38:31 --> syntax error, unexpected 'except' (T_STRING), expecting ']'
#0 C:\xampp\htdocs\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
#3 C:\xampp\htdocs\system\Config\Services.php(322): spl_autoload_call('Config\\Filters')
#4 C:\xampp\htdocs\system\Config\BaseService.php(123): CodeIgniter\Config\Services::filters(NULL, false)
#5 C:\xampp\htdocs\system\Config\Services.php(317): CodeIgniter\Config\BaseService::getSharedInstance('filters', NULL)
#6 C:\xampp\htdocs\system\CodeIgniter.php(367): CodeIgniter\Config\Services::filters()
#7 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2021-09-19 13:38:35 --> syntax error, unexpected 'except' (T_STRING), expecting ']'
#0 C:\xampp\htdocs\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
#3 C:\xampp\htdocs\system\Config\Services.php(322): spl_autoload_call('Config\\Filters')
#4 C:\xampp\htdocs\system\Config\BaseService.php(123): CodeIgniter\Config\Services::filters(NULL, false)
#5 C:\xampp\htdocs\system\Config\Services.php(317): CodeIgniter\Config\BaseService::getSharedInstance('filters', NULL)
#6 C:\xampp\htdocs\system\CodeIgniter.php(367): CodeIgniter\Config\Services::filters()
#7 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2021-09-19 13:41:05 --> <br>The encoded file <b>C:\xampp\htdocs\app\Filters\CheckLogin.php</b> is corrupt.
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-09-19 13:42:32 --> <br>The encoded file <b>C:\xampp\htdocs\app\Filters\CheckLogin.php</b> is corrupt.
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-09-19 13:45:14 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '' at line 1
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('select user_typ...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('select user_typ...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('select user_typ...')
#3 C:\xampp\htdocs\app\Controllers\Home.php(0): CodeIgniter\Database\BaseConnection->query('select user_typ...')
#4 C:\xampp\htdocs\system\CodeIgniter.php(918): App\Controllers\Home->index()
#5 C:\xampp\htdocs\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-09-19 13:46:00 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '' at line 1
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('select user_typ...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('select user_typ...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('select user_typ...')
#3 C:\xampp\htdocs\app\Controllers\Home.php(0): CodeIgniter\Database\BaseConnection->query('select user_typ...')
#4 C:\xampp\htdocs\system\CodeIgniter.php(918): App\Controllers\Home->index()
#5 C:\xampp\htdocs\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-09-19 13:51:09 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '' at line 1
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('select user_typ...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('select user_typ...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('select user_typ...')
#3 C:\xampp\htdocs\app\Controllers\Home.php(0): CodeIgniter\Database\BaseConnection->query('select user_typ...')
#4 C:\xampp\htdocs\system\CodeIgniter.php(918): App\Controllers\Home->index()
#5 C:\xampp\htdocs\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-09-19 13:51:13 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '' at line 1
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('select user_typ...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('select user_typ...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('select user_typ...')
#3 C:\xampp\htdocs\app\Controllers\Home.php(0): CodeIgniter\Database\BaseConnection->query('select user_typ...')
#4 C:\xampp\htdocs\system\CodeIgniter.php(918): App\Controllers\Home->index()
#5 C:\xampp\htdocs\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-09-19 13:54:04 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '' at line 1
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('select user_typ...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('select user_typ...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('select user_typ...')
#3 C:\xampp\htdocs\app\Controllers\Home.php(0): CodeIgniter\Database\BaseConnection->query('select user_typ...')
#4 C:\xampp\htdocs\system\CodeIgniter.php(918): App\Controllers\Home->index()
#5 C:\xampp\htdocs\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-09-19 13:54:46 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '' at line 1
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('select user_typ...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('select user_typ...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('select user_typ...')
#3 C:\xampp\htdocs\app\Controllers\Home.php(0): CodeIgniter\Database\BaseConnection->query('select user_typ...')
#4 C:\xampp\htdocs\system\CodeIgniter.php(918): App\Controllers\Home->index()
#5 C:\xampp\htdocs\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-09-19 13:55:49 --> Unable to connect to the database.
#0 C:\xampp\htdocs\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\app\Controllers\Home.php(0): CodeIgniter\Database\BaseConnection->query('select user_typ...')
#2 C:\xampp\htdocs\system\CodeIgniter.php(918): App\Controllers\Home->index()
#3 C:\xampp\htdocs\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#4 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-09-19 13:55:49 --> Uncaught ErrorException: print_r(): Property access is not allowed yet in C:\xampp\htdocs\app\Views\errors\html\error_exception.php:0
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'print_r(): Prop...', 'C:\\xampp\\htdocs...', 0, Array)
#1 C:\xampp\htdocs\app\Views\errors\html\error_exception.php(0): print_r(Object(App\Controllers\Home), true)
#2 C:\xampp\htdocs\system\Debug\Exceptions.php(308): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Database\Exceptions\DatabaseException), 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Database\Exceptions\DatabaseException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-09-19 13:55:56 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '' at line 1
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('select user_typ...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('select user_typ...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('select user_typ...')
#3 C:\xampp\htdocs\app\Controllers\Home.php(0): CodeIgniter\Database\BaseConnection->query('select user_typ...')
#4 C:\xampp\htdocs\system\CodeIgniter.php(918): App\Controllers\Home->index()
#5 C:\xampp\htdocs\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-09-19 13:56:57 --> Class '\CodeIgniter\Database\MySQL\Connection' not found
#0 C:\xampp\htdocs\system\Database\Config.php(115): CodeIgniter\Database\Database->load(Array, 'default')
#1 C:\xampp\htdocs\app\Models\tank_auth\Users.php(32): CodeIgniter\Database\Config::connect()
#2 C:\xampp\htdocs\system\Database\ModelFactory.php(109): App\Models\Users->__construct(NULL)
#3 C:\xampp\htdocs\system\Database\ModelFactory.php(40): CodeIgniter\Database\ModelFactory::createClass('App\\Models\\User...', NULL)
#4 C:\xampp\htdocs\system\Common.php(774): CodeIgniter\Database\ModelFactory::get('App\\Models\\tank...', true, NULL)
#5 C:\xampp\htdocs\app\Libraries\Tank_auth.php(0): model('App\\Models\\tank...')
#6 C:\xampp\htdocs\app\Controllers\Home.php(0): App\Libraries\Tank_auth->__construct()
#7 C:\xampp\htdocs\system\CodeIgniter.php(890): App\Controllers\Home->__construct()
#8 C:\xampp\htdocs\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#9 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2021-09-19 13:57:03 --> Class '\CodeIgniter\Database\MySQL\Connection' not found
#0 C:\xampp\htdocs\system\Database\Config.php(115): CodeIgniter\Database\Database->load(Array, 'default')
#1 C:\xampp\htdocs\app\Models\tank_auth\Users.php(32): CodeIgniter\Database\Config::connect()
#2 C:\xampp\htdocs\system\Database\ModelFactory.php(109): App\Models\Users->__construct(NULL)
#3 C:\xampp\htdocs\system\Database\ModelFactory.php(40): CodeIgniter\Database\ModelFactory::createClass('App\\Models\\User...', NULL)
#4 C:\xampp\htdocs\system\Common.php(774): CodeIgniter\Database\ModelFactory::get('App\\Models\\tank...', true, NULL)
#5 C:\xampp\htdocs\app\Libraries\Tank_auth.php(0): model('App\\Models\\tank...')
#6 C:\xampp\htdocs\app\Controllers\Home.php(0): App\Libraries\Tank_auth->__construct()
#7 C:\xampp\htdocs\system\CodeIgniter.php(890): App\Controllers\Home->__construct()
#8 C:\xampp\htdocs\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#9 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2021-09-19 14:00:07 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '' at line 1
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('select user_typ...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('select user_typ...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('select user_typ...')
#3 C:\xampp\htdocs\app\Controllers\Home.php(0): CodeIgniter\Database\BaseConnection->query('select user_typ...')
#4 C:\xampp\htdocs\system\CodeIgniter.php(918): App\Controllers\Home->index()
#5 C:\xampp\htdocs\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-09-19 14:01:55 --> Division by zero
#0 C:\xampp\htdocs\app\Views\office\dashboard\js_file.php(0): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Division by zer...', 'C:\\xampp\\htdocs...', 0, Array)
#1 C:\xampp\htdocs\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\system\Common.php(1119): CodeIgniter\View\View->render('office/dashboar...', Array, true)
#3 C:\xampp\htdocs\app\Views\template\footer.php(0): view('office/dashboar...')
#4 C:\xampp\htdocs\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\system\Common.php(1119): CodeIgniter\View\View->render('template/footer...', Array, true)
#6 C:\xampp\htdocs\app\Views\template\index.php(0): view('template/footer...')
#7 C:\xampp\htdocs\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#8 C:\xampp\htdocs\system\Common.php(1119): CodeIgniter\View\View->render('template/index', Array, true)
#9 C:\xampp\htdocs\app\Controllers\Dashboard.php(0): view('template/index', Array)
#10 C:\xampp\htdocs\system\CodeIgniter.php(918): App\Controllers\Dashboard->index()
#11 C:\xampp\htdocs\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#12 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#14 {main}
CRITICAL - 2021-09-19 14:01:56 --> Division by zero
#0 C:\xampp\htdocs\app\Views\office\dashboard\js_file.php(0): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Division by zer...', 'C:\\xampp\\htdocs...', 0, Array)
#1 C:\xampp\htdocs\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\system\Common.php(1119): CodeIgniter\View\View->render('office/dashboar...', Array, true)
#3 C:\xampp\htdocs\app\Views\template\footer.php(0): view('office/dashboar...')
#4 C:\xampp\htdocs\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\system\Common.php(1119): CodeIgniter\View\View->render('template/footer...', Array, true)
#6 C:\xampp\htdocs\app\Views\template\index.php(0): view('template/footer...')
#7 C:\xampp\htdocs\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#8 C:\xampp\htdocs\system\Common.php(1119): CodeIgniter\View\View->render('template/index', Array, true)
#9 C:\xampp\htdocs\app\Controllers\Dashboard.php(0): view('template/index', Array)
#10 C:\xampp\htdocs\system\CodeIgniter.php(918): App\Controllers\Dashboard->index()
#11 C:\xampp\htdocs\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#12 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#14 {main}
CRITICAL - 2021-09-19 14:02:11 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '' at line 1
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('select user_typ...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('select user_typ...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('select user_typ...')
#3 C:\xampp\htdocs\app\Controllers\Home.php(0): CodeIgniter\Database\BaseConnection->query('select user_typ...')
#4 C:\xampp\htdocs\system\CodeIgniter.php(918): App\Controllers\Home->index()
#5 C:\xampp\htdocs\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-09-19 14:02:14 --> Division by zero
#0 C:\xampp\htdocs\app\Views\office\dashboard\js_file.php(0): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Division by zer...', 'C:\\xampp\\htdocs...', 0, Array)
#1 C:\xampp\htdocs\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\system\Common.php(1119): CodeIgniter\View\View->render('office/dashboar...', Array, true)
#3 C:\xampp\htdocs\app\Views\template\footer.php(0): view('office/dashboar...')
#4 C:\xampp\htdocs\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\system\Common.php(1119): CodeIgniter\View\View->render('template/footer...', Array, true)
#6 C:\xampp\htdocs\app\Views\template\index.php(0): view('template/footer...')
#7 C:\xampp\htdocs\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#8 C:\xampp\htdocs\system\Common.php(1119): CodeIgniter\View\View->render('template/index', Array, true)
#9 C:\xampp\htdocs\app\Controllers\Dashboard.php(0): view('template/index', Array)
#10 C:\xampp\htdocs\system\CodeIgniter.php(918): App\Controllers\Dashboard->index()
#11 C:\xampp\htdocs\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#12 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#14 {main}
CRITICAL - 2021-09-19 14:02:14 --> Division by zero
#0 C:\xampp\htdocs\app\Views\office\dashboard\js_file.php(0): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Division by zer...', 'C:\\xampp\\htdocs...', 0, Array)
#1 C:\xampp\htdocs\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\system\Common.php(1119): CodeIgniter\View\View->render('office/dashboar...', Array, true)
#3 C:\xampp\htdocs\app\Views\template\footer.php(0): view('office/dashboar...')
#4 C:\xampp\htdocs\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\system\Common.php(1119): CodeIgniter\View\View->render('template/footer...', Array, true)
#6 C:\xampp\htdocs\app\Views\template\index.php(0): view('template/footer...')
#7 C:\xampp\htdocs\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#8 C:\xampp\htdocs\system\Common.php(1119): CodeIgniter\View\View->render('template/index', Array, true)
#9 C:\xampp\htdocs\app\Controllers\Dashboard.php(0): view('template/index', Array)
#10 C:\xampp\htdocs\system\CodeIgniter.php(918): App\Controllers\Dashboard->index()
#11 C:\xampp\htdocs\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#12 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#14 {main}
CRITICAL - 2021-09-19 14:03:01 --> Column 'user_id' cannot be null
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('INSERT INTO `us...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\system\Common.php(1281): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\app\Controllers\Auth.php(0): trail(1, 'Logout', 'Logout Auth', Array)
#6 C:\xampp\htdocs\system\CodeIgniter.php(918): App\Controllers\Auth->logout()
#7 C:\xampp\htdocs\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-09-19 14:03:20 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '' at line 1
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('select user_typ...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('select user_typ...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('select user_typ...')
#3 C:\xampp\htdocs\app\Controllers\Home.php(0): CodeIgniter\Database\BaseConnection->query('select user_typ...')
#4 C:\xampp\htdocs\system\CodeIgniter.php(918): App\Controllers\Home->index()
#5 C:\xampp\htdocs\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-09-19 14:31:32 --> Table 'FAWE_db.c:\xampp\htdocs\writable\session' doesn't exist
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `data`\nF...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `data`\nF...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `data`\nF...')
#3 C:\xampp\htdocs\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `data`\nF...', Array, false)
#4 C:\xampp\htdocs\system\Session\Handlers\DatabaseHandler.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->read('jm7ppm9mgodfs0c...')
#6 C:\xampp\htdocs\system\Session\Session.php(1005): session_start()
#7 C:\xampp\htdocs\system\Session\Session.php(240): CodeIgniter\Session\Session->startSession()
#8 C:\xampp\htdocs\system\Config\Services.php(788): CodeIgniter\Session\Session->start()
#9 C:\xampp\htdocs\system\Config\BaseService.php(123): CodeIgniter\Config\Services::session(Object(Config\App), false)
#10 C:\xampp\htdocs\system\Config\Services.php(769): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#11 C:\xampp\htdocs\app\Libraries\Tank_auth.php(35): CodeIgniter\Config\Services::session()
#12 C:\xampp\htdocs\app\Controllers\Auth.php(0): App\Libraries\Tank_auth->__construct()
#13 C:\xampp\htdocs\system\CodeIgniter.php(890): App\Controllers\Auth->__construct()
#14 C:\xampp\htdocs\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#15 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#16 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#17 {main}
CRITICAL - 2021-09-19 14:33:46 --> Table 'FAWE_db.c:\xampp\htdocs\writable\session' doesn't exist
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `data`\nF...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `data`\nF...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `data`\nF...')
#3 C:\xampp\htdocs\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `data`\nF...', Array, false)
#4 C:\xampp\htdocs\system\Session\Handlers\DatabaseHandler.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->read('imaodjfo16fmbhr...')
#6 C:\xampp\htdocs\system\Session\Session.php(1005): session_start()
#7 C:\xampp\htdocs\system\Session\Session.php(240): CodeIgniter\Session\Session->startSession()
#8 C:\xampp\htdocs\system\Config\Services.php(788): CodeIgniter\Session\Session->start()
#9 C:\xampp\htdocs\system\Config\BaseService.php(123): CodeIgniter\Config\Services::session(Object(Config\App), false)
#10 C:\xampp\htdocs\system\Config\Services.php(769): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#11 C:\xampp\htdocs\app\Libraries\Tank_auth.php(35): CodeIgniter\Config\Services::session()
#12 C:\xampp\htdocs\app\Controllers\Auth.php(0): App\Libraries\Tank_auth->__construct()
#13 C:\xampp\htdocs\system\CodeIgniter.php(890): App\Controllers\Auth->__construct()
#14 C:\xampp\htdocs\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#15 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#16 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#17 {main}
CRITICAL - 2021-09-19 14:33:54 --> Table 'FAWE_db.c:\xampp\htdocs\writable\session' doesn't exist
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `data`\nF...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `data`\nF...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `data`\nF...')
#3 C:\xampp\htdocs\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `data`\nF...', Array, false)
#4 C:\xampp\htdocs\system\Session\Handlers\DatabaseHandler.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->read('ueohjhd6qgq6pqp...')
#6 C:\xampp\htdocs\system\Session\Session.php(1005): session_start()
#7 C:\xampp\htdocs\system\Session\Session.php(240): CodeIgniter\Session\Session->startSession()
#8 C:\xampp\htdocs\system\Config\Services.php(788): CodeIgniter\Session\Session->start()
#9 C:\xampp\htdocs\system\Config\BaseService.php(123): CodeIgniter\Config\Services::session(Object(Config\App), false)
#10 C:\xampp\htdocs\system\Config\Services.php(769): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#11 C:\xampp\htdocs\app\Libraries\Tank_auth.php(35): CodeIgniter\Config\Services::session()
#12 C:\xampp\htdocs\app\Controllers\Auth.php(0): App\Libraries\Tank_auth->__construct()
#13 C:\xampp\htdocs\system\CodeIgniter.php(890): App\Controllers\Auth->__construct()
#14 C:\xampp\htdocs\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#15 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#16 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#17 {main}
CRITICAL - 2021-09-19 14:34:00 --> Table 'FAWE_db.c:\xampp\htdocs\writable\session' doesn't exist
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `data`\nF...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `data`\nF...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `data`\nF...')
#3 C:\xampp\htdocs\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `data`\nF...', Array, false)
#4 C:\xampp\htdocs\system\Session\Handlers\DatabaseHandler.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->read('jfcetm2r2do22bp...')
#6 C:\xampp\htdocs\system\Session\Session.php(1005): session_start()
#7 C:\xampp\htdocs\system\Session\Session.php(240): CodeIgniter\Session\Session->startSession()
#8 C:\xampp\htdocs\system\Config\Services.php(788): CodeIgniter\Session\Session->start()
#9 C:\xampp\htdocs\system\Config\BaseService.php(123): CodeIgniter\Config\Services::session(Object(Config\App), false)
#10 C:\xampp\htdocs\system\Config\Services.php(769): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#11 C:\xampp\htdocs\app\Libraries\Tank_auth.php(35): CodeIgniter\Config\Services::session()
#12 C:\xampp\htdocs\app\Controllers\Auth.php(0): App\Libraries\Tank_auth->__construct()
#13 C:\xampp\htdocs\system\CodeIgniter.php(890): App\Controllers\Auth->__construct()
#14 C:\xampp\htdocs\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#15 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#16 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#17 {main}
CRITICAL - 2021-09-19 14:35:06 --> Table 'FAWE_db.c:\xampp\htdocs\writable\session' doesn't exist
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `data`\nF...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `data`\nF...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `data`\nF...')
#3 C:\xampp\htdocs\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `data`\nF...', Array, false)
#4 C:\xampp\htdocs\system\Session\Handlers\DatabaseHandler.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->read('me0h7qrlp3j2ru9...')
#6 C:\xampp\htdocs\system\Session\Session.php(1005): session_start()
#7 C:\xampp\htdocs\system\Session\Session.php(240): CodeIgniter\Session\Session->startSession()
#8 C:\xampp\htdocs\system\Config\Services.php(788): CodeIgniter\Session\Session->start()
#9 C:\xampp\htdocs\system\Config\BaseService.php(123): CodeIgniter\Config\Services::session(Object(Config\App), false)
#10 C:\xampp\htdocs\system\Config\Services.php(769): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#11 C:\xampp\htdocs\app\Libraries\Tank_auth.php(35): CodeIgniter\Config\Services::session()
#12 C:\xampp\htdocs\app\Controllers\Auth.php(0): App\Libraries\Tank_auth->__construct()
#13 C:\xampp\htdocs\system\CodeIgniter.php(890): App\Controllers\Auth->__construct()
#14 C:\xampp\htdocs\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#15 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#16 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#17 {main}
CRITICAL - 2021-09-19 14:37:04 --> Table 'FAWE_db.c:\xampp\htdocs\writable\session' doesn't exist
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `data`\nF...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `data`\nF...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `data`\nF...')
#3 C:\xampp\htdocs\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `data`\nF...', Array, false)
#4 C:\xampp\htdocs\system\Session\Handlers\DatabaseHandler.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->read('8i9klso7ipags3k...')
#6 C:\xampp\htdocs\system\Session\Session.php(1005): session_start()
#7 C:\xampp\htdocs\system\Session\Session.php(240): CodeIgniter\Session\Session->startSession()
#8 C:\xampp\htdocs\system\Config\Services.php(788): CodeIgniter\Session\Session->start()
#9 C:\xampp\htdocs\system\Config\BaseService.php(123): CodeIgniter\Config\Services::session(Object(Config\App), false)
#10 C:\xampp\htdocs\system\Config\Services.php(769): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#11 C:\xampp\htdocs\app\Libraries\Tank_auth.php(35): CodeIgniter\Config\Services::session()
#12 C:\xampp\htdocs\app\Controllers\Auth.php(0): App\Libraries\Tank_auth->__construct()
#13 C:\xampp\htdocs\system\CodeIgniter.php(890): App\Controllers\Auth->__construct()
#14 C:\xampp\htdocs\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#15 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#16 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#17 {main}
CRITICAL - 2021-09-19 14:37:51 --> Table 'FAWE_db.c:\xampp\htdocs\writable\session' doesn't exist
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `data`\nF...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `data`\nF...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `data`\nF...')
#3 C:\xampp\htdocs\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `data`\nF...', Array, false)
#4 C:\xampp\htdocs\system\Session\Handlers\DatabaseHandler.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->read('18cf95fpv5egvki...')
#6 C:\xampp\htdocs\system\Session\Session.php(1005): session_start()
#7 C:\xampp\htdocs\system\Session\Session.php(240): CodeIgniter\Session\Session->startSession()
#8 C:\xampp\htdocs\system\Config\Services.php(788): CodeIgniter\Session\Session->start()
#9 C:\xampp\htdocs\system\Config\BaseService.php(123): CodeIgniter\Config\Services::session(Object(Config\App), false)
#10 C:\xampp\htdocs\system\Config\Services.php(769): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#11 C:\xampp\htdocs\app\Libraries\Tank_auth.php(35): CodeIgniter\Config\Services::session()
#12 C:\xampp\htdocs\app\Controllers\Auth.php(0): App\Libraries\Tank_auth->__construct()
#13 C:\xampp\htdocs\system\CodeIgniter.php(890): App\Controllers\Auth->__construct()
#14 C:\xampp\htdocs\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#15 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#16 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#17 {main}
CRITICAL - 2021-09-19 14:38:45 --> Table 'FAWE_db.c:\xampp\htdocs\writable\session' doesn't exist
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `data`\nF...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `data`\nF...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `data`\nF...')
#3 C:\xampp\htdocs\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `data`\nF...', Array, false)
#4 C:\xampp\htdocs\system\Session\Handlers\DatabaseHandler.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->read('td2f40m3t5s7oen...')
#6 C:\xampp\htdocs\system\Session\Session.php(1005): session_start()
#7 C:\xampp\htdocs\system\Session\Session.php(240): CodeIgniter\Session\Session->startSession()
#8 C:\xampp\htdocs\system\Config\Services.php(788): CodeIgniter\Session\Session->start()
#9 C:\xampp\htdocs\system\Config\BaseService.php(123): CodeIgniter\Config\Services::session(Object(Config\App), false)
#10 C:\xampp\htdocs\system\Config\Services.php(769): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#11 C:\xampp\htdocs\app\Libraries\Tank_auth.php(35): CodeIgniter\Config\Services::session()
#12 C:\xampp\htdocs\app\Controllers\Auth.php(0): App\Libraries\Tank_auth->__construct()
#13 C:\xampp\htdocs\system\CodeIgniter.php(890): App\Controllers\Auth->__construct()
#14 C:\xampp\htdocs\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#15 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#16 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#17 {main}
CRITICAL - 2021-09-19 14:38:55 --> Table 'FAWE_db.c:\xampp\htdocs\writable\session' doesn't exist
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `data`\nF...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `data`\nF...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `data`\nF...')
#3 C:\xampp\htdocs\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `data`\nF...', Array, false)
#4 C:\xampp\htdocs\system\Session\Handlers\DatabaseHandler.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->read('8f8jis0lqo1kr4j...')
#6 C:\xampp\htdocs\system\Session\Session.php(1005): session_start()
#7 C:\xampp\htdocs\system\Session\Session.php(240): CodeIgniter\Session\Session->startSession()
#8 C:\xampp\htdocs\system\Config\Services.php(788): CodeIgniter\Session\Session->start()
#9 C:\xampp\htdocs\system\Config\BaseService.php(123): CodeIgniter\Config\Services::session(Object(Config\App), false)
#10 C:\xampp\htdocs\system\Config\Services.php(769): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#11 C:\xampp\htdocs\app\Libraries\Tank_auth.php(35): CodeIgniter\Config\Services::session()
#12 C:\xampp\htdocs\app\Controllers\Auth.php(0): App\Libraries\Tank_auth->__construct()
#13 C:\xampp\htdocs\system\CodeIgniter.php(890): App\Controllers\Auth->__construct()
#14 C:\xampp\htdocs\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#15 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#16 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#17 {main}
CRITICAL - 2021-09-19 14:40:50 --> Table 'FAWE_db.c:\xampp\htdocs\writable\session' doesn't exist
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `data`\nF...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `data`\nF...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `data`\nF...')
#3 C:\xampp\htdocs\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `data`\nF...', Array, false)
#4 C:\xampp\htdocs\system\Session\Handlers\DatabaseHandler.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->read('krs7n0f22buqa7i...')
#6 C:\xampp\htdocs\system\Session\Session.php(1005): session_start()
#7 C:\xampp\htdocs\system\Session\Session.php(240): CodeIgniter\Session\Session->startSession()
#8 C:\xampp\htdocs\system\Config\Services.php(788): CodeIgniter\Session\Session->start()
#9 C:\xampp\htdocs\system\Config\BaseService.php(123): CodeIgniter\Config\Services::session(Object(Config\App), false)
#10 C:\xampp\htdocs\system\Config\Services.php(769): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#11 C:\xampp\htdocs\app\Libraries\Tank_auth.php(35): CodeIgniter\Config\Services::session()
#12 C:\xampp\htdocs\app\Controllers\Auth.php(0): App\Libraries\Tank_auth->__construct()
#13 C:\xampp\htdocs\system\CodeIgniter.php(890): App\Controllers\Auth->__construct()
#14 C:\xampp\htdocs\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#15 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#16 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#17 {main}
CRITICAL - 2021-09-19 14:42:16 --> Unable to connect to the database.
#0 C:\xampp\htdocs\system\Session\Handlers\DatabaseHandler.php(142): CodeIgniter\Database\BaseConnection->initialize()
#1 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->open('C:\\xampp\\htdocs...', 'ci_session')
#2 C:\xampp\htdocs\system\Session\Session.php(1005): session_start()
#3 C:\xampp\htdocs\system\Session\Session.php(240): CodeIgniter\Session\Session->startSession()
#4 C:\xampp\htdocs\system\Config\Services.php(788): CodeIgniter\Session\Session->start()
#5 C:\xampp\htdocs\system\Config\BaseService.php(123): CodeIgniter\Config\Services::session(Object(Config\App), false)
#6 C:\xampp\htdocs\system\Config\Services.php(769): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#7 C:\xampp\htdocs\app\Libraries\Tank_auth.php(35): CodeIgniter\Config\Services::session()
#8 C:\xampp\htdocs\app\Controllers\Auth.php(0): App\Libraries\Tank_auth->__construct()
#9 C:\xampp\htdocs\system\CodeIgniter.php(890): App\Controllers\Auth->__construct()
#10 C:\xampp\htdocs\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#11 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2021-09-19 14:42:29 --> Table 'FAWE_db.c:\xampp\htdocs\writable\session' doesn't exist
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `data`\nF...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `data`\nF...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `data`\nF...')
#3 C:\xampp\htdocs\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `data`\nF...', Array, false)
#4 C:\xampp\htdocs\system\Session\Handlers\DatabaseHandler.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->read('2etodrp3o1dl47b...')
#6 C:\xampp\htdocs\system\Session\Session.php(1005): session_start()
#7 C:\xampp\htdocs\system\Session\Session.php(240): CodeIgniter\Session\Session->startSession()
#8 C:\xampp\htdocs\system\Config\Services.php(788): CodeIgniter\Session\Session->start()
#9 C:\xampp\htdocs\system\Config\BaseService.php(123): CodeIgniter\Config\Services::session(Object(Config\App), false)
#10 C:\xampp\htdocs\system\Config\Services.php(769): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#11 C:\xampp\htdocs\app\Libraries\Tank_auth.php(35): CodeIgniter\Config\Services::session()
#12 C:\xampp\htdocs\app\Controllers\Auth.php(0): App\Libraries\Tank_auth->__construct()
#13 C:\xampp\htdocs\system\CodeIgniter.php(890): App\Controllers\Auth->__construct()
#14 C:\xampp\htdocs\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#15 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#16 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#17 {main}
CRITICAL - 2021-09-19 14:44:01 --> Table 'FAWE_db.c:\xampp\htdocs\writable\session' doesn't exist
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `data`\nF...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `data`\nF...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `data`\nF...')
#3 C:\xampp\htdocs\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `data`\nF...', Array, false)
#4 C:\xampp\htdocs\system\Session\Handlers\DatabaseHandler.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->read('57f42pq5fmqct93...')
#6 C:\xampp\htdocs\system\Session\Session.php(1005): session_start()
#7 C:\xampp\htdocs\system\Session\Session.php(240): CodeIgniter\Session\Session->startSession()
#8 C:\xampp\htdocs\system\Config\Services.php(788): CodeIgniter\Session\Session->start()
#9 C:\xampp\htdocs\system\Config\BaseService.php(123): CodeIgniter\Config\Services::session(Object(Config\App), false)
#10 C:\xampp\htdocs\system\Config\Services.php(769): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#11 C:\xampp\htdocs\app\Libraries\Tank_auth.php(35): CodeIgniter\Config\Services::session()
#12 C:\xampp\htdocs\app\Controllers\Auth.php(0): App\Libraries\Tank_auth->__construct()
#13 C:\xampp\htdocs\system\CodeIgniter.php(890): App\Controllers\Auth->__construct()
#14 C:\xampp\htdocs\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#15 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#16 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#17 {main}
CRITICAL - 2021-09-19 14:46:22 --> Table 'FAWE_db.c:\xampp\htdocs\writable\session' doesn't exist
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `data`\nF...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `data`\nF...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `data`\nF...')
#3 C:\xampp\htdocs\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `data`\nF...', Array, false)
#4 C:\xampp\htdocs\system\Session\Handlers\DatabaseHandler.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->read('v0rvqgqg2jdrb7l...')
#6 C:\xampp\htdocs\system\Session\Session.php(1005): session_start()
#7 C:\xampp\htdocs\system\Session\Session.php(240): CodeIgniter\Session\Session->startSession()
#8 C:\xampp\htdocs\system\Config\Services.php(788): CodeIgniter\Session\Session->start()
#9 C:\xampp\htdocs\system\Config\BaseService.php(123): CodeIgniter\Config\Services::session(Object(Config\App), false)
#10 C:\xampp\htdocs\system\Config\Services.php(769): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#11 C:\xampp\htdocs\app\Libraries\Tank_auth.php(35): CodeIgniter\Config\Services::session()
#12 C:\xampp\htdocs\app\Controllers\Auth.php(0): App\Libraries\Tank_auth->__construct()
#13 C:\xampp\htdocs\system\CodeIgniter.php(890): App\Controllers\Auth->__construct()
#14 C:\xampp\htdocs\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#15 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#16 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#17 {main}
CRITICAL - 2021-09-19 14:46:31 --> Table 'FAWE_db.c:\xampp\htdocs\writable\session' doesn't exist
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `data`\nF...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `data`\nF...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `data`\nF...')
#3 C:\xampp\htdocs\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `data`\nF...', Array, false)
#4 C:\xampp\htdocs\system\Session\Handlers\DatabaseHandler.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->read('q2blnlq9876bjnj...')
#6 C:\xampp\htdocs\system\Session\Session.php(1005): session_start()
#7 C:\xampp\htdocs\system\Session\Session.php(240): CodeIgniter\Session\Session->startSession()
#8 C:\xampp\htdocs\system\Config\Services.php(788): CodeIgniter\Session\Session->start()
#9 C:\xampp\htdocs\system\Config\BaseService.php(123): CodeIgniter\Config\Services::session(Object(Config\App), false)
#10 C:\xampp\htdocs\system\Config\Services.php(769): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#11 C:\xampp\htdocs\app\Libraries\Tank_auth.php(35): CodeIgniter\Config\Services::session()
#12 C:\xampp\htdocs\app\Controllers\Auth.php(0): App\Libraries\Tank_auth->__construct()
#13 C:\xampp\htdocs\system\CodeIgniter.php(890): App\Controllers\Auth->__construct()
#14 C:\xampp\htdocs\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#15 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#16 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#17 {main}
CRITICAL - 2021-09-19 14:46:36 --> Table 'FAWE_db.c:\xampp\htdocs\writable\session' doesn't exist
#0 C:\xampp\htdocs\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `data`\nF...')
#1 C:\xampp\htdocs\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `data`\nF...')
#2 C:\xampp\htdocs\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `data`\nF...')
#3 C:\xampp\htdocs\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `data`\nF...', Array, false)
#4 C:\xampp\htdocs\system\Session\Handlers\DatabaseHandler.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->read('meja2u96reg39kq...')
#6 C:\xampp\htdocs\system\Session\Session.php(1005): session_start()
#7 C:\xampp\htdocs\system\Session\Session.php(240): CodeIgniter\Session\Session->startSession()
#8 C:\xampp\htdocs\system\Config\Services.php(788): CodeIgniter\Session\Session->start()
#9 C:\xampp\htdocs\system\Config\BaseService.php(123): CodeIgniter\Config\Services::session(Object(Config\App), false)
#10 C:\xampp\htdocs\system\Config\Services.php(769): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#11 C:\xampp\htdocs\app\Libraries\Tank_auth.php(35): CodeIgniter\Config\Services::session()
#12 C:\xampp\htdocs\app\Controllers\Auth.php(0): App\Libraries\Tank_auth->__construct()
#13 C:\xampp\htdocs\system\CodeIgniter.php(890): App\Controllers\Auth->__construct()
#14 C:\xampp\htdocs\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#15 C:\xampp\htdocs\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#16 C:\xampp\htdocs\index.php(45): CodeIgniter\CodeIgniter->run()
#17 {main}
